variables.tf

```diff
@@ -380,3 +380,10 @@ variable "block_device_mappings" {
   description = "Block Device Mapping Object"
 }
 ##################
+
+#ResourceTagSpecification#
+variable "should_tag_volumes" {
+  type        = bool
+  default     = false
```

Comment:
this will set the value of the attribute always to false, please change this and test

Reply:
resolved